test/externalTests/zeppelin.sh

```diff
@@ -69,6 +69,9 @@ function zeppelin_test
     sed -i "s|it(\('reverts \)|it.skip(\1|g" math/SignedSafeMath.test.js
     sed -i "s|it(\('reverts \)|it.skip(\1|g" structs/EnumerableSet.behavior.js
     popd
+    pushd test/proxy/
+    sed -i "s|it(\('proxy admin cannot \)|it.skip(\1|g" transparent/TransparentUpgradeableProxy.behaviour.js
```

Comment:
Looks like this one is just a single test so we can put in the full name. The `reverts` above was done like that because it covers multiple tests. I'd err on the side of being specific where possible.
```suggestion
    sed -i "s|it(\('proxy admin cannot call delegated functions')\)|it.skip(\1|g" transparent/TransparentUpgradeableProxy.behaviour.js
```